warehouse/admin/views/checks.py

```diff
@@ -56,32 +56,36 @@ def get_check(request):
 
 
 @view_config(
-    route_name="admin.checks.run_backfill",
+    route_name="admin.checks.run_evaluation",
     permission="admin",
     request_method="POST",
     uses_session=True,
     require_methods=False,
     require_csrf=True,
 )
-def run_backfill(request):
+def run_evaluation(request):
     check = get_check_by_name(request.db, request.matchdict["check_name"])
-    num_objects = 10000
 
     if check.state not in (MalwareCheckState.enabled, MalwareCheckState.evaluation):
         request.session.flash(
-            f"Check must be in 'enabled' or 'evaluation' state to run a backfill.",
+            f"Check must be in 'enabled' or 'evaluation' state to manually execute.",
             queue="error",
         )
         return HTTPSeeOther(
             request.route_path("admin.checks.detail", check_name=check.name)
         )
 
-    request.session.flash(
-        f"Running {check.name} on {num_objects} {check.hooked_object.value}s!",
-        queue="success",
-    )
+    if check.check_type == MalwareCheckType.event_hook:
+        num_objects = 10000
```

Comment:
Ultra nit: Maybe parametrize this somewhere (either as a module-level constant or as something loaded into the application-level config via `includeme`)?